datadog-log/src/logger.rs

```diff
@@ -0,0 +1,361 @@
+// Copyright 2025-Present Datadog, Inc. https://www.datadoghq.com/
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::writers::{FileWriter, NoopWriter, StdoutWriter};
+use ddcommon_ffi::Error;
+use once_cell::sync::Lazy;
```

Comment:
Since we're not currently covering no_std scenarios wouldn't be better to use std::sync::OnceCell. It basically provide the same functionality and we'll remove a dependency.

Reply:
fair call, I was under impression that we are already using `once_cell` as dependency.